### PR TITLE
Prompts user if there are tasks assigned to a patient when user is tr…

### PR DESCRIPTION
**Pull request by kkeejjuunn**
When user is trying to delete a patient by id or name, the system will check if there are any tasks are assigned to this patient. There will be a double confirmation in case that user accidentally delete the wrong patient. Related to issue #134 

If there are tasks assigned to this patient, all these tasks will be show so the user can check whether these tasks have been performed.  
image

If user confirms to delete a patient, the tasks assigned to this patient will be deleted as well.

### CODE REVIEW

File: src/main/java/duke/command/DeletePatientCommand.java [hunk 1]
```diff
@@ -51,33 +54,47 @@ public DeletePatientCommand(String deletedPatientInfo) throws DukeException {
...
+        try {
+            patientTask = patientTaskList.getPatientTask(patientToBeDeleted.getID());
+            ArrayList<Task> tempTask = new ArrayList<>();
+            for (PatientTask temppatientTask : patientTask) {
+                tempTask.add(taskManager.getTask(temppatientTask.getTaskID()));
+            }
+            ui.patientTaskFound(patientToBeDeleted, patientTask, tempTask);
+            toDelete = ui.confirmPatientToBeDeleted(patientToBeDeleted, true);
+        } catch (Exception e) {
+            toDelete = ui.confirmPatientToBeDeleted(patientToBeDeleted,false);
```
Comment (HUANGXUANKUN) [inline src/main/java/duke/command/DeletePatientCommand.java]: You might considered to use a method provided by PatientTask class to get the list of PatientTasks assigned to a Patient.

**Comment by kkeejjuunn** [comment]
> Some minor typos
> image

@WEIFENG-NUSCEG Can correct the typo based on Xuankun's feedback? I am calling your patientTaskFound method in ui class.

File: src/main/java/duke/command/DeletePatientCommand.java (hunk 1 above)
Comment (kkeejjuunn) [reply src/main/java/duke/command/DeletePatientCommand.java]: Actually I am using the method provided by PatientTask class to get the list of PatientTasks at line 80. From line 81 to 84, I am trying to get the list of tasks from taskManager that linked to each PatientTask I get from line 80.

**Comment by WEIFENG-NUSCEG** [comment]
> > Some minor typos
> > image
> 
> @WEIFENG-NUSCEG Can correct the typo based on Xuankun's feedback? I am calling your patientTaskFound method in ui class.

This problem will be solved in my PR Thx for notice

File: src/main/java/duke/command/DeletePatientCommand.java (hunk 1 above)
Comment (HUANGXUANKUN) [reply src/main/java/duke/command/DeletePatientCommand.java]: Yes you are right. Sorry for misunderstanding the code. There is no redundant code here.

Comment (HUANGXUANKUN) [reply src/main/java/duke/command/DeletePatientCommand.java]: Yes you are right. Sorry for misunderstanding the code. There is no redundant code here.

**Pull request merged**